raiden-ts/docs-source/SDK-Development.md

```diff
@@ -171,13 +170,15 @@ The `Raiden` client class is the entrypoint for the public API. The focus of thi
+
+For public interface simplicity and usability, a subset of the actions is exposed through an `events$` observable as an alternative to duplicating almost everything on explicit reimplementations using `EventEmitter`s or any other more common approach. `channels$` list is also exposed through a public observable mapped from `state$`. `state$` is publicly exposed so users willing to pass an initial state and manage the state persistency themselves (instead of the recommended `localStorage` approach) can do so by listening on state changes on this observable, but value's format must not be relied on and considered implementation detail and subject to change.
 
 Despite all of that, state machine correctness and simplicity is a priority over public API class implementation complexity. Therefore, non-state changing (aka read-only) method may be implemented inside Raiden directly, or in pure separate utility functions when needed.
 
 ## Actions
 
-We follow the [Flux Standard Action (`FSA`)](https://github.com/redux-utilities/flux-standard-action) pattern when creating actions. They should always sit on the same module as the reducer which handles them. The usual redux approach requires creating/changing up to 4 places to create a new action: an `enum` of action types, a `interface` for the action object, a `factory function` and a `tagged union` of actions, to have proper reducer type safety.
+We follow the [Flux Standard Action (`FSA`)](https://github.com/redux-utilities/flux-standard-action) pattern when creating actions. They should always be defined in the same module as the reducer which handles them. The usual redux approach requires creating/changing up to 4 places to create a new action: an `enum` of action types, a `interface` for the action object, a `factory function` and a `tagged union` of actions, to have proper reducer type safety.
 
 The `typesafe-actions` help a lot with that by allowing us to define a new action in a single place: defining an `action-creator` object. For FSA schema, we use [`createStandardAction`](https://github.com/piotrwitek/typesafe-actions#2-fsa-compliant-actions) helper. It'll create an object in the module's action submodule, and this action-creator both acts directly as action factory, and contain all metadata needed for the actions: `getType(action)` gives the `string literal` used as `type` tag for the action (allowing to easily narrow type on `switch (action.type)` statements), `ActionType<typeof action>` is used to get the interface/type/schema for the action, and `ActionType<typeof Actions>` also allows to use `import * as Actions` mappings to get a tagged union of all exported actions in a module.
```

Comment:
We don't use this lib anymore, and instead I've reimplemented its functionality in `src/utils/actions.ts`, but using our `io-ts` codecs to make it even more powerful.

Reply:
There're several new facilities which aren't mentioned yet: `createReducer` helps creating reducers and adding `handle`rs for type-safe functions. Not sure if `partialCombineReducers` is mentioned either